### ISSUE TITLE
k and m value

### ISSUE THREAD

**Issue by mrgil123** [facebook/prophet#2059]
Good Afternoon
Firstly I want to say thank you for creating robust forecasting machine!
Recently I studied about Prophet
I confuse.. 
How did Prophet calculate k and m value?
I really need your help

**Comment by sirbentham** [comment]
Hey, maybe this is the answer you are searching for:
https://github.com/facebook/prophet/blob/ba9a5a2c6e2400206017a5ddfd71f5042da9f65b/python/prophet/forecaster.py#L992-L1013

**Comment by tcuongd** [comment]
Yep, that's how the initial values for `k` and `m` are determined, then they get optimised by the fitting procedure, which is defined in stan here: https://github.com/facebook/prophet/blob/main/python/stan/unix/prophet.stan
